src/EluvioLive.js

```diff
@@ -64,84 +57,106 @@ class EluvioLive {
-	const abiTenant = fs.readFileSync(path.resolve(__dirname, "../contracts/v3/BaseTenantSpace.abi"));
-
-	var tenantInfo = {};
-
-	var m = await this.client.ContentObjectMetadata({
-	  libraryId,
-	  objectId,
-	  metadataSubtree: "/public/asset_metadata",
-	  resolveLinks: true,
-	  resolveIncludeSource: true,
-	  resolveIgnoreErrors: true,
-	  linkDepthLimit: 5
-	});
-
-	tenantInfo.marketplaces = {};
-	var warns = [];
-
-	for (var key in m.marketplaces) {
-	  tenantInfo.marketplaces[key] = {};
-	  tenantInfo.marketplaces[key].items = {};
-	  for (var i in m.marketplaces[key].info.items) {
-
-		const item = m.marketplaces[key].info.items[i];
-
-		const sku = item.sku;
-
-		if (item.nft_template == "") {
-		  warns.push("No NFT Template sku: " + sku);
-		  continue;
-		}
-
-		tenantInfo.marketplaces[key].items[sku] = {};
-		tenantInfo.marketplaces[key].items[sku].name = item.name;
-		tenantInfo.marketplaces[key].items[sku].description = item.description;
-		tenantInfo.marketplaces[key].items[sku].mint_cauth = item.nft_template.mint.cauth_id;
-		tenantInfo.marketplaces[key].items[sku].nft_addr = item.nft_template.nft.address;
-		tenantInfo.marketplaces[key].items[sku].templateTotalSupply = item.nft_template.nft.total_supply;
-		tenantInfo.marketplaces[key].items[sku].nft_template = item.nft_template["."].source;
-
-		if (cauth && cauth != item.nft_template.mint.cauth_id) {
-		  warns.push("Wrong cauth_id sku: " + sku);
-		}
-
-		if (item.nft_template.nft.address === "") {
-		  warns.push("No NFT address sku: " + sku);
-		} else {
-		  // Check NFT contract parameters
-		  const nftInfo = await this.NftShow({addr: item.nft_template.nft.address, mintHelper});
-		  tenantInfo.marketplaces[key].items[sku].nftCap = nftInfo.cap;
-		  tenantInfo.marketplaces[key].items[sku].nftMinted = nftInfo.minted;
-		  tenantInfo.marketplaces[key].items[sku].nftTotalSupply = nftInfo.totalSupply;
-		  tenantInfo.marketplaces[key].items[sku].nftName = nftInfo.name;
-		  tenantInfo.marketplaces[key].items[sku].proxy = nftInfo.owner;
-		  tenantInfo.marketplaces[key].items[sku].proxy = nftInfo.proxy;
-		  tenantInfo.marketplaces[key].items[sku].firstTokenUri = nftInfo.firstTokenUri;
-		  tenantInfo.marketplaces[key].items[sku].defHoldSecs = nftInfo.defHoldSecs;
-
-		  if (nftInfo.cap != item.nft_template.nft.total_supply) {
-			warns.push("NFT cap mismatch sku: " + sku);
-		  }
-		  if (nftInfo.proxyInfo != null && (nftInfo.owner != nftInfo.proxyInfo.owner)) {
-			warns.push("NFT owner not proxy owner: " + sku);
-		  }
-		  if (nftInfo.warns.length > 0) {
-			warns.push(...nftInfo.warns);
-		  }
-		}
-	  }
-	}
-
-	tenantInfo.sites = {};
-	tenantInfo.warns = warns;
-
-	return tenantInfo;
+  async TenantShow({
+    tenantId,
+    libraryId,
+    objectId,
+    eventId,
+    marketplaceId,
+    cauth,
+    mintHelper,
+  }) {
+    const abiNft = fs.readFileSync(
+      path.resolve(__dirname, "../contracts/v3/ElvTradable.abi")
```

Comment:
I think this has to be ElvTradableLocal - can you pls try?  Can be a separate commit